src/docs/ftw-content/how-to-change-ftw-bundled-ui-translations/index.md

```diff
@@ -1,26 +1,35 @@
 ---
-title: How to change FTW UI texts and translations
-slug: how-to-change-ftw-ui-texts-and-translations
-updated: 2019-05-21
-category: ftw-styling
+title: How to change FTW bundled translations
+slug: how-to-change-ftw-bundled-translations
+updated: 2022-05-13
+category: ftw-content
 ingress:
-  This guide describes how to change the User Interface (UI) texts and
-  translations in Flex Template for Web (FTW).
+  This guide describes how to change the bundled translations in Flex
+  Template for Web (FTW).
 published: true
 ---
 
 The Flex Template for Web supports having a single language for the UI.
 Supported languages are English, French and Spanish, English being used
 by default. For information about changing the language, see the
-[Changing the language](#changing-the-language) section below.
+[Changing the language](/ftw/how-to-change-ftw-language) article.
 
 We are using the [React Intl](https://github.com/yahoo/react-intl)
 library to translate UI texts and to format dates, numbers, and money
 values.
 
+_**Note:** Starting in 2022-05, FTW template translations can also be
+modified in Flex Console. If you want to implement this feature into
+your pre-version XX.XX FTW template, you can see the necessary
```

Comment:
Related to this, I think the next version are:
- FTW-daily v8.5
- FTW-hourly v10.5
- FTW-product: v9.2

aka minor version update